Mathlib/Algebra/Algebra/Basic.lean

```diff
@@ -248,6 +248,7 @@ end FieldDivisionRing
 end algebraMap
 
 /-- Creating an algebra from a morphism to the center of a semiring. -/
+@[reducible]
 def RingHom.toAlgebra' {R S} [CommSemiring R] [Semiring S] (i : R →+* S)
```

Comment:
These instance are nasty enough anyway that probably making them reducible doesn't make much difference. Could you benchmark this pair of changes separately?

Reply:
Sure. Let me see this impact together first. But I'll do it either way.

Reply:
Actually taking a look at the benchmark results here, do you still want this split?

Reply:
Eh, already had it queued up #9952

Reply:
I just restored them to `default`. I only realize now that I was operating off a previous attempt and didn't need this.